app/controllers/projects_controller.rb

```diff
@@ -28,16 +28,19 @@ def create
   end
 
   def update
-    if @project.update(project_params)
+    if update_project.success?
       redirect_to projects_path, notice: "Update Successful"
     else
       render :edit, status: :unprocessable_entity
     end
   end
 
   def destroy
-    @project.destroy
-    redirect_to projects_path, notice: "Project destroyed"
+    if delete_project.success?
+      redirect_to projects_path, notice: "Project destroyed"
+    else
+
+    end
```

Comment:
Don't use if block without else
```suggestion
    redirect_to projects_path, notice: "Project destroyed" if delete_project.success?
```